src/components/Header/TopMainNav/TopMainNavIllustration/TopMainNavAblyLogo.tsx

```diff
@@ -13,7 +13,7 @@ export const TopMainNavAblyLogo = ({ href = '/' }: { href?: string }) => {
       {width > 1024 ? (
         <StaticImage src="/images/icons/ably-docs-logo.svg" />
       ) : (
-        <StaticImage className="h-32" src="/images/icons/ably-logo.svg" />
+        <StaticImage className="h-32" src="/images/icons/ably-docs-logo-mobile.svg" />
```

Comment:
Slight scope creep, but can you make sure both these images don't resize by giving them a fixed width and height as well?

Reply:
@MymmiJ  updated and got approval from Lenka for the position of the logo